rust/main/config/mainnet_config.json

```diff
@@ -7209,6 +7209,65 @@
       },
       "gnosisSafeTransactionServiceUrl": "https://safe-transaction-sonic.safe.global/"
     },
+    "starknet": {
+      "blockExplorers": [
+        {
+          "apiUrl": "https://voyager.online/api",
+          "family": "voyager",
+          "name": "Starknet Explorer",
+          "url": "https://voyager.online/"
+        }
+      ],
+      "blocks": {
+        "confirmations": 1,
+        "estimateBlockTime": 30,
+        "reorgPeriod": 1
+      },
+      "chainId": "0x534e5f4d41494e",
+      "deployer": {
+        "name": "Abacus Works",
+        "url": "https://www.hyperlane.xyz"
+      },
+      "displayName": "Starknet",
+      "domainId": 3047739480,
+      "gasCurrencyCoinGeckoId": "ethereum",
+      "name": "starknet",
+      "nativeToken": {
+        "decimals": 18,
+        "name": "Ether",
+        "symbol": "ETH",
+        "denom": "0x049D36570D4e46f48e99674bd3fcc84644DdD6b96F7C741B1562B82f9e004dC7"
+      },
+      "protocol": "starknet",
+      "rpcUrls": [
+        {
+          "http": "https://starknet-mainnet.public.blastapi.io"
+        },
+        {
+          "http": "https://rpc.starknet.lava.build:443"
+        }, 
+        {
+          "http": "https://starknet.api.onfinality.io/public"
+        },
+        {
+          "http": "https://api.zan.top/public/starknet-mainnet"
+        }
+      ],
+      "defaultHook": "0x011f16e1a49ec2646e92933b34984cd218eaea2be70b645323d678207b35b22c",
+      "defaultIsm": "0x0493aaa0f86386541c7ffe0b78e1c929a46fb01ca71021e5327d0576309d3777",
+      "mailbox": "0x03ebe74a2a21cae6b204126dad1317b1f3e9eccd5b1636a81606df9051d87e1b",
+      "merkleTreeHook": "0x0669054378957087a28778b7ca38f55c222875dc6f35d5c58e51d1d22702f9d5",
+      "noopIsm": "0x017370983b9a003483d8ab7e422bfe2ea2713111fef860b28286340cc6f6d834",
+      "protocolFee": "0x011f16e1a49ec2646e92933b34984cd218eaea2be70b645323d678207b35b22c",
+      "testRecipient": "0x03efc4ec979176a4b13daa7858719386a0947f4d4429ed9dd6d7664fb462200e",
+      "validatorAnnounce": "0x07dab388ed5aad7f4f7a9669a9bb64ea7fe3ae4357c7a98d20785a2808e24b63",
+      "interchainGasPaymaster": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
also my bad for not catching this is missing in the registry pr https://github.com/hyperlane-xyz/hyperlane-registry/commit/62851e3206253bee648d3c1bc280eb38455f737d#diff-f0be03968efa8f2c9e47c8b089a3d20aaa0ce5ff5c41c4281648e8f3faddb5d5